xtevent/xtevent.sthlp

```diff
@@ -115,19 +115,19 @@ and {opt overidpost()} (See below).
 {phang2} {opt pre} is the number of pre-event periods where anticipation effects are allowed. With {opt window}, {opt pre} is 0.
 
 {phang2} {opt post} is the number of post-event periods where policy effects are allowed. With {opt window}, {opt post} is the number
-of periods after the event minus 2.
+of periods after the event (not including the period for the event, e.g. event time = 0).
 
 {phang2} {opt overidpre} is the number of pre-event periods for an overidentification test of pre-trends. With {opt window}, {opt overidpre}
 is the number of periods before the event.
 
 {phang2} {opt overidpost} is the number of post-event periods for an overidentification test of effects leveling off. With {opt window},
-{opt overidpost} is 2.
+{opt overidpost} is 1.

```

Comment:
In -xteventtest- it is taking a value of 2
https://github.com/JMSLab/xtevent/blob/d2111190adaa71e6af655c0d2ceb1dbe722a2631/xtevent/xteventtest.ado#L78